src/enclave-agent/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "enclave-agent"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+protobuf = { version = "=2.14.0" }
+bytes = "0.4.11"
+libc = "0.2.58"
+byteorder = "1.3.2"
+log = "0.4.11"
+simple-logging = "2.0.2"
+nix = "0.23.0"
+ttrpc = { git = "https://github.com/haosanzi/ttrpc-rust", features = ["async"] }
+ctrlc = { version = "3.0", features = ["termination"] }
+
+# Async runtime
+tokio = { version = "1.14.0", features = ["full"] }
+async-trait = "0.1.42"
+
+image-rs = { git = "https://github.com/zhiwei-intel-h/image-rs", branch = "dev-occlum-adaption", features = ["occlum_feature"], default-features = false }
```

Comment:
Please use `https://github.com/confidential-containers/image-rs` since it has supported occlum unionfs

Reply:
> Please use `https://github.com/confidential-containers/image-rs` since it has supported occlum unionfs

I have replaced it with `image-rs = { git = "https://github.com/confidential-containers/image-rs", features = ["occlum_feature"], default-features = false }`.